README.md

```diff
@@ -8,29 +8,69 @@ Memory caching with the following focus:
 - High concurrent throughput
```

Comment:
I notice https://pkg.go.dev/github.com/graxinc/cache is out of date for some reason 🤔

Reply:
I just hit https://pkg.go.dev/github.com/graxinc/cache@main and I think that sorted it out. Probably no (public) fetches of the latest version to refresh things before that.